pkg/reconciler/buildrun/buildrun_test.go

```diff
@@ -127,7 +127,7 @@ var _ = Describe("Reconcile BuildRun", func() {
 
 				// initialize a TaskRun, we need this to fake the existence of a Tekton TaskRun
 				taskRunSample = ctl.DefaultTaskRunWithStatus(taskRunName, buildRunName, ns, corev1.ConditionTrue, "Succeeded")
-
+				taskRunSample.Spec.ServiceAccountName = "test"
```

Comment:
I am wondering, could we place this `"test"` service account name in `DefaultTaskRunWithStatus` directly?